djhtml/modes.py

```diff
@@ -385,10 +385,10 @@ class DjJS(DjTXT):
         r"//.*",
         r"/\*",
         r"[$\w-]+:",
-        r'"(?:\\.|[^\\"])*"',  # "string"
-        r"'(?:\\.|[^\\'])*'",  # 'string'
-        r"`(?:\\.|[^\\`])*`",  # `string`
-        r"/(?:\\.|[^\\/ \n])*/",  # /string/ (but without spaces)
+        r'"(?:\\.|[^"])*"',  # "string"
+        r"'(?:\\.|[^'])*'",  # 'string'
+        r"`(?:\\.|[^`])*`",  # `string`
+        r"/(?:\\.|[^ /\n])(?:\\.|[^/\n])*(?:\\.|[^ /\n])/",  # /string/
```

Comment:
This will only match regular expressions of length 2 or more, so will not match `/a/`.